markdown-toc.el

```diff
@@ -101,12 +101,18 @@
        (-repeat it sym)
        (s-join "" it)))
 
+(defconst markdown-toc--protection-symbol "09876543214b825dc642cb6eb9a060e54bf8d69288fbee49041234567890"
```

Comment:
It's just a long random string, nothing magic? I am curious about this. @ardumont